tests/conftest.py

```diff
@@ -136,8 +136,10 @@ def rat_board_fx() -> RatBoard:
 
 
 @pytest.fixture()
-def bot_fx(configuration_fx):
-    return MockBot(nickname="mock_mecha3[BOT]", mecha_config=configuration_fx)
+def bot_fx(configuration_fx, galaxy_fx):
+    mock_bot = MockBot(nickname="mock_mecha3[BOT]", mecha_config=configuration_fx)
+    mock_bot._galaxy = galaxy_fx
```

Comment:
`mock_bot.galaxy` has a setter. use it.

Reply:
There is a warning that made me thinking using setter is discouraged. =]

Reply:
not entirely sure why there is a warning message there, but i think it had the rational that it shouldn't just be set without good reason (and thus should be logged).

That said, its bad practice to bypass a setter and access private attributes of an object from outside said object.
Yes this is a test, but i don't think this specific case warrants bypassing the setter.
*objectively this should probably be included in the MechaClient constructor but thats out of scope*

Reply:
Noted.